### ISSUE TITLE
Views counter is stuck!

### ISSUE THREAD

**Issue by Thomas-George-T** [wesky93/views#1]
I believe the views counter is stuck, not sure if it's a temporary thing.

Please could you look into it.

Thanks for hosting the clone!

**Comment by wesky93** [comment]
I'm debug that issue! fix it ASAP

**Comment by wesky93** [comment]
I've solved the problem. 
However, some badges that were created before the problem was solved may still be problematic. 
Because it's already cached in github cdn. 
So if the badge you're using doesn't work, add a query as shown below to make it look like a new url.
if your orgin badege url is this
```
![ViewCount](https://views.whatilearened.today/views/github/Thomas-George-Thomas/Thomas-George-Thomas.svg)
```
then add some query string like this!
```
![ViewCount](https://views.whatilearened.today/views/github/Thomas-George-Thomas/Thomas-George-Thomas.svg?cache=remove)
```
It will work normally.
